kratos/python/add_dof_utilities_to_python.cpp

```diff
@@ -0,0 +1,75 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define_python.h"
+#include "includes/model_part.h"
+#include "python/add_dof_utilities_to_python.h"
+#include "utilities/dof_utilities/dof_array_utilities.h"
+#include "utilities/dof_utilities/block_build_dof_array_utility.h"
+#include "utilities/dof_utilities/elimination_build_dof_array_utility.h"
+
+PYBIND11_MAKE_OPAQUE(Kratos::DofArrayUtilities::SlaveToMasterDofsMap); // prevents pybind11 from trying to convert the unordered_map into a dict
+
+namespace Kratos::Python
+{
+
+void AddDofUtilitiesToPython(pybind11::module& m)
+{
+    namespace py = pybind11;
+
+    py::class_<BlockBuildDofArrayUtility>(m, "BlockBuildDofArrayUtility")
+        .def_static("SetUpDofArray", [](const ModelPart& rModelPart, BlockBuildDofArrayUtility::DofsArrayType& rDofArray){BlockBuildDofArrayUtility::SetUpDofArray(rModelPart, rDofArray);})
+        .def_static("SetUpDofArray", [](const ModelPart& rModelPart, BlockBuildDofArrayUtility::DofsArrayType& rDofArray, const unsigned int EchoLevel){BlockBuildDofArrayUtility::SetUpDofArray(rModelPart, rDofArray, EchoLevel);})
+        .def_static("SetUpDofArray", [](const ModelPart& rModelPart, BlockBuildDofArrayUtility::DofsArrayType& rDofArray, const unsigned int EchoLevel, const bool CheckReactionDofs){BlockBuildDofArrayUtility::SetUpDofArray(rModelPart, rDofArray, EchoLevel, CheckReactionDofs);})
+    ;
```

Comment:
I would put `py::arg` for all the arguements, so it is readable from python side. [eg. `py::arg("echo_level") = 0`]